packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -54,11 +54,30 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                     data-heading="{{:: continent }}"
                 >
                     <div class="container-fluid px-0">
+                        <p>Global regions ... Oles ipsum...</p>
                         <div class="row">
                             <oui-select-picker
                                 class="d-inline-block col-md-6 col-xl-3 my-3"
                                 data-ng-repeat="(location, datacenters) in datacentersLocation track by location"
-                                data-ng-if="datacenters.length"
+                                data-ng-if="datacenters.length && location !== 'Madrid'"
```

Comment:
`Madrid` check is only for mocking data ? If yes, add a TODO to not forget to remove it

Reply:
I have removed it here : https://github.com/ovh/manager/pull/10523